answers-Misbah.js

```diff
@@ -0,0 +1,89 @@
+// Programming Basics: For Loops
+// These exercises are designed for practising "for" loops. Print all your results to the console.
+
+// Addition. Write a program to add up the numbers 1 to 20.
+let numList = 0
+
+for (let i = 1; i <= 20; i++){  
+   numList = numList + 1;
+   console.log(`${numList}`);
+}  
+
+// There are i bottles of beer on the wall. Write a program that will output, "There is one bottle of beer on the wall." "There are two bottles of beer on the wall" up until there are five bottles.
+let beer = 0
+
+for (let i = 1; i <= 5; i++){  
```

Comment:
think about using if condition and i

Reply:
why would i use if?? please comment only if there is something wrong with the code!!
THANKS

Reply:
we use if else to manage more than one case
if one bottle:
`There is one bottle of beer on the wall.`

if more than one 
"There are two bottles of beer on the wall"
...